api/sentry/api_sentry.py

```diff
@@ -60,9 +73,29 @@ def releases(self):
                 'projects/{0}/firefox-ios/releases/'
                 '?&project={1}&statsPeriod=1d'
                 '&environment=Production'
-            ).format(self.organization_slug, self.project_id)
+            ).format(self.organization_slug, self.sentry_project_id)
         )
 
+    # API: Releases (with filtering)
+    # /organizations/{{organization_slug}}/releases/
+    # ?adoptionStages=1&environment={{environment}}&project={{project_id}}
+    # &query=release.package:{{package}}&status=open&summaryStatsPeriod=24h
+    # &sort=adoption&adoptionStages=adopted
+    def releases_v2(self):
```

Comment:
recommend rename to `get_releases_with_adoption` or `get_releases_with_filtering`

Reply:
`def releases_v2()` supersedes the `def releases()` because `v2` includes a parameter to query Fenix releases.